src/Sulu/Bundle/MediaBundle/Resources/js/components/ImageFocusPoint/FocusPoint.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {Icon} from 'sulu-admin-bundle/components';
+import type {Point} from './types';
+import focusPointStyles from './focusPoint.scss';
+
+const ARROW_UP_ICON = 'arrow-up';
+
+type Props = {
+    size: number,
+    value: Point,
+    active: boolean,
+    onClick?: (value: Point) => void,
+    showArrow: boolean,
+    arrowDirection: number,
```

Comment:
Why don't you use a union type with strings here? Similar to what you did for the `Grid` component? I guess that would be easier to read when you are writing code.

Reply:
Then we could maybe also merge the two previous attributes?